### DISCUSSION TITLE
i will Suppresses  "opentelemetry.proto.collector.metrics.v1.MetricsService/Export"

### DISCUSSION THREAD

**Discussion by 478467337** [open-telemetry/opentelemetry-java-instrumentation#5478] [Q&A]
export OTEL_JAVAAGENT_EXCLUDE_CLASSES=io.opentelemetry.exporter.otlp.internal.*,io.opentelemetry.*

**Comment by trask** [comment]
hi @478467337, I'm not following, can you elaborate?

**Reply by 478467337** [reply]
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/suppressing-instrumentation.md
i want Suppresses all instrumentation for io.opentelemetry.exporter.otlp.metrics.MarshalerMetricsServiceGrpc

**Reply by trask** [reply]
do you want to suppress metrics? metrics should be off by default (until metrics SDK is declared stable), unless you opt in using `OTEL_METRICS_EXPORTER`
